### PR TITLE
only pull open vulnerabilities for sonarqube2hdf as closed vulnerabilities

### PR DESCRIPTION
**Pull request by blake-burton**
Issue
===

Sonarqube2hdf failing with bad request caused by pulling closed vulnerabilities. The source pulls **all** the vulnerabilities, then for each vulnerability pulls the source code associated with it.  Vulnerabilities which are closed  causes an error because the file it was in no longer exists. This is the cause of the  BAD REQUEST is requesting source that no longer exists.

Solution
======

Only pull open and reopened vulnerabilities, this will ensure that the files which the vulnerabilities exists will still exist.

### CODE REVIEW

**Comment by blake-burton** [comment]
@Amndeep7 @camdenmoors @charleshu-8 Would it be possible to squeeze in a review for this?

**Comment by Amndeep7** [comment]
Thanks for the PR, I'll take a look soon

**Review by Amndeep7 — APPROVED** [review]

File: libs/hdf-converters/src/sonarqube-mapper.ts [hunk 1]
```diff
@@ -130,6 +130,7 @@ export class SonarQubeResults {
           params: {
             componentKeys: this.projectId,
             types: 'VULNERABILITY',
+            statuses: 'OPEN,REOPENED',
```
Comment (Amndeep7) [inline libs/hdf-converters/src/sonarqube-mapper.ts:133]:
```
Statuses

After creation, issues flow through a lifecycle, taking one of the following statuses:

    Open - set by SonarQube on new issues
    Confirmed - set manually to indicate that the issue is valid
    Resolved - set manually to indicate that the next analysis should Close the issue
    Reopened - set automatically by SonarQube when a Resolved issue hasn't actually been corrected
    Closed - set automatically by SonarQube for automatically created issues.
```

Shouldn't we actually be looking at all of these statuses aside from 'Closed'?

**Review by Amndeep7 — CHANGES_REQUESTED** [review]

File: libs/hdf-converters/src/sonarqube-mapper.ts (hunk 1 above)
Comment (blake-burton) [reply libs/hdf-converters/src/sonarqube-mapper.ts:133]: Good call! Confirmed and  use case i have that cause this was only closed issues.

**Review by Amndeep7 — APPROVED** [review]

**Pull request merged**